### ISSUE TITLE
blank image

### ISSUE THREAD

**Issue by sangjun04** [Maclory/SPSR#37]
I am training with my custom data. I have successfully trained on custom data and the validation result looks good.
However, when I run the exact same model on a test set, all images are black. I have checked pixel values, and only one channel among three has values between 0 and 20 even after the normalization.

Any clue what might have caused this issue?

**Comment by Maclory** [comment]
Hi, I think this may be caused by incorrect preprocessing of testing images. You can check whether the input pattern of the testing set is similar to the pattern of other images.

**Comment by sangjun04** [comment]
To make sure the model is working, I used the same image from the validation set for testing.

**Comment by Maclory** [comment]
Does the model work well on the validation set?

**Comment by sangjun04** [comment]
The validation result created during training looks valid.

**Comment by stdinR** [comment]
Hello, I want to ask about custom training.
Since you said, you used your own dataset. Is HR size in your dataset different from default (128) given by the author?
I found when I changed the HR_size = 128 to my own HR size (which is 480) in train_spsr.json, the training didn't work.
However, when I used the default HR_size (128) without any change, it could train.
This makes me wonder would this impact the training result?

**Comment by leoyjTu** [comment]
> Hello, I want to ask about custom training. Since you said, you used your own dataset. Is HR size in your dataset different from default (128) given by the author? I found when I changed the HR_size = 128 to my own HR size (which is 480) in train_spsr.json, the training didn't work. However, when I used the default HR_size (128) without any change, it could train. This makes me wonder would this impact the training result?

I'm having the same problem. Have you solved it now?
